apps/cookbook/src/app/examples/modal-example/modal-example-configuration/modal-example-configuration.component.ts

```diff
@@ -40,7 +40,7 @@ export class ModalExampleConfigurationComponent {
   @Input() openFullHeight: boolean;
   @Output() openFullHeightChange = new EventEmitter<boolean>();
 
-  @HostBinding('class.checkbox-xs') true; // Extra small checkboxes
+  @HostBinding('class.checkbox-xs') isCheckboxXs: boolean = true; // Extra small checkboxes
```

Comment:
@MartinSkovsen

Reply:
See DM from @jakobe. Also I've had the same trouble with that on a different PR in a different component (just setting the hostbinding to true).